frontend/orders-pane.js

```diff
@@ -798,6 +805,11 @@ class OrdersPane extends HTMLElement {
 		} else {
 			for (let e of shadow.querySelectorAll("#score_switches input")) e.disabled = true;
 		}
+		if (g_data.date >= 26 && (g_data.date - 26) % 6 == 0) {
+			shadow.querySelector("#tab_game").classList.add("alert");
```

Comment:
Already getting some mileage out of this alert class tagging!

Reply:
Yep! :D